### PR TITLE
chore: add survey invitation button to homescreen

### PR DESCRIPTION
**Pull request by rschlaefli**

### CODE REVIEW

File: apps/auth/tsconfig.json [hunk 1]
```diff
@@ -16,9 +20,18 @@
...
+      ]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
-  "exclude": ["node_modules"]
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "../../packages/shared-components/src/hooks/useStickyState.ts"
```
Comment (rschlaefli) [inline apps/auth/tsconfig.json]:
```suggestion
```

File: apps/frontend-pwa/src/pages/index.tsx [hunk 2]
```diff
@@ -45,8 +50,16 @@ type LocalMicroLearningType = Partial<MicroLearning> & {
...
 const Index = function () {
+  const router = useRouter()
   const t = useTranslations()
 
+  const { value: hasSeenSurvey, setValue: setHasSeenSurvey } = useStickyState(
+    'hasSeenSurvey',
+    false
+  )
+
+  console.log(hasSeenSurvey)
```
Comment (rschlaefli) [inline apps/frontend-pwa/src/pages/index.tsx]:
```suggestion

```

Comment (sjschlapbach) [inline apps/frontend-pwa/src/pages/index.tsx]:
```suggestion
```

File: apps/frontend-pwa/src/pages/index.tsx [hunk 3]
```diff
@@ -214,6 +226,27 @@ const Index = function () {
         className="flex flex-col gap-4 md:w-full md:max-w-xl md:p-8 md:mx-auto md:border md:rounded"
         data-cy="homepage"
       >
+        {hasSeenSurvey === 'false' && (
```
Comment (sjschlapbach) [inline apps/frontend-pwa/src/pages/index.tsx:228]: Is this value also set to a string immediately on initialization?

**Review by sjschlapbach — APPROVED** [review]
LGTM

File: apps/frontend-pwa/src/pages/index.tsx [hunk 4]
```diff
@@ -45,8 +50,15 @@ type LocalMicroLearningType = Partial<MicroLearning> & {
 }
 
 const Index = function () {
+  const router = useRouter()
   const t = useTranslations()
 
+  const { value: hasSeenSurvey, setValue: setHasSeenSurvey } = useStickyState(
+    'hasSeenSurvey',
+    false
```
Comment (sjschlapbach) [inline apps/frontend-pwa/src/pages/index.tsx]:
```suggestion
    'false'
```

**Pull request merged**